integration-tests/api/__tests__/admin/order-edit/order-edit.js

```diff
@@ -2572,13 +2570,15 @@ describe("/admin/order-edits", () => {
               ]),
             }),
           ]),
-          discount_total: 2000,
+          // rounding issue since we are allocating 1/3 of the discount to one item and 2/3 to the other item where both cost 10
+          // resulting in adjustment amounts like: 1333...
+          discount_total: 2001,
+          total: 1099,
```

Comment:
note: change in total amounts in this test

Reply:
Do you know what change resulted in this happening now when it didn't occur previously?

Reply:
We decorate [OE totals via order service now](https://github.com/medusajs/medusa/pull/3108#discussion_r1086611377) which uses the new totals service, previously we used "old" totals directly from OE service.

Reply:
should we do something about this rounding or did you agree to something?